docs/deployment.md

```diff
@@ -132,91 +127,18 @@ If you don't want to set environment variables, [access keys can be stored local
 
 See [creating s3 buckets](./create_bucket.md) for more information on how to create a bucket.
 
-The following s3 buckets should be created (replacing prefix with whatever you'd like, generally your organization/DAAC's name):
+The following s3 bucket should be created (replacing prefix with whatever you'd like, generally your organization/DAAC's name):
```

Comment:
Minor but this doesn't really need to be a list any more, since we're only requiring 1 bucket.

Reply:
I think it just looks nice having it stand out, but if you strongly disagree I will change it!